client/interface.go

```diff
@@ -0,0 +1,28 @@
+package client
+
+import (
+	"context"
+	"encoding"
+	"time"
+)
+
+// Client represents the drand Client interface.
+type Client interface {
+	// TextMarshaler (the 'MarshalText' method) allows serialization of the client
+	// state for subsequent restoration. The marshalled value can be passed to `New`
+	// to restore Client state.
+	encoding.TextMarshaler
+
+	// Get returns a the randomness at `round` or an error.
+	Get(ctx context.Context, round uint64) (Result, error)
+
+	// RoundAt will return the most recent round of randomness that will be available
+	// at time for the current client.
+	RoundAt(time time.Time) uint64
```

Comment:
it can be calculated locally once the group info is fetched in the constructor. it's purely a function of the parameters of the group this client is connected to, and not an rpc call.